### PR TITLE
GameSettings: Add patch for Monster High: Ghoul Spirit (SAOE78/SAOEVZ)

### PR DESCRIPTION
**Pull request by smurf3tte**
The first call to GXCopyDisp() corrupts game data, but on real hardware it isn't observed thanks to the data cache. Skipping the call works too, preventing a crash on boot.

This patch is enabled default.

### CODE REVIEW

**Comment by leoetlino** [comment]
Marking this as "do not merge" until #9317 is merged.

File: Data/Sys/GameSettings/SAOE78.ini
Comment (leoetlino) [inline Data/Sys/GameSettings/SAOE78.ini:7]: Please enable this by default, with

```ini
[OnFrame_Enabled]
$Fix crash on boot
```

**Review by leoetlino — CHANGES_REQUESTED** [review]

File: Data/Sys/GameSettings/SAOE78.ini
Comment (smurf3tte) [reply Data/Sys/GameSettings/SAOE78.ini:7]: Done!

**Comment by smurf3tte** [comment]
With help from @JMC47 I was able to add support for the version of the game published by Little Orbit (SAOEVZ). They also confirmed that the PAL release does not need this fix. Thanks JMC47!

**Comment by JMC47** [comment]
PAL doesn't need a patch.  Tested the patch for the v1.01 version of the game.

**Review by leoetlino — APPROVED** [review]

**Comment by ytrezq** [comment]
@smurf3tte would it be possible to produce a patch for an Icache access when going in game menus, for this game I own https://bugs.dolphin-emu.org/issues/8037?

**Pull request merged**
